plugins/fetchai-crypto/tests/data/fetchai_private_key.txt

```diff
@@ -0,0 +1 @@
+6d56fd47e98465824aa85dfe620ad3dbf092b772abc6c6a182e458b5c56ad13b
```

Comment:
Please remove and change test to use temp dir

Reply:
TODO

Reply:
done